src/status_im/contexts/settings/privacy_and_security/profile_picture/view.cljs

```diff
@@ -0,0 +1,124 @@
+(ns status-im.contexts.settings.privacy-and-security.profile-picture.view
+  (:require
+    [quo.core :as quo]
+    [quo.theme]
+    [status-im.constants :as constants]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(def pictures-visibility->label
+  {constants/profile-pictures-visibility-everyone      (i18n/label :t/everyone)
+   constants/profile-pictures-visibility-contacts-only (i18n/label :t/contacts)
+   constants/profile-pictures-visibility-none          (i18n/label :t/no-one)})
+
+(def pictures-show-to->label
```

Comment:
Should we also use similar name for others profile picture? Like `profile-pictures-show-from-everyone`

Reply:
Currently we have,

```
profile-pictures-visibility-everyone
profile-pictures-visibility-contacts-only
etc...
```

Maybe we can use
```
profile-picture-show-from-everyone
profile-pictures-show-from-contacts-only 
etc...
```

This will be similar to `profile-pictures-show-to-everyone` and more clear. wdyt?

Reply:
Hey @Parveshdhull, does this commit improves things in your opinion? I renamed things to be consistent https://github.com/status-im/status-mobile/pull/20671/commits/48d13e129e8d76ddc950add7c05aaa8eff5bc39d